### DISCUSSION TITLE
开启通过服务器转发消息后，一直断开重连

### DISCUSSION THREAD

**Discussion by OcrSu** [xfgryujk/blivechat#197] [提问] [answered]
我是在本地的Openwrt搭建的Docker服务器。
在开启 通过服务器转发消息 之前，报错是500代码。

开启后则是一直重连，请大佬帮忙看看该怎么解决。

开启后：
![image](https://github.com/xfgryujk/blivechat/assets/75876201/6405a6b1-34ae-4471-9210-f2f9051da21a)

开启前：
![image](https://github.com/xfgryujk/blivechat/assets/75876201/15600623-a485-4190-ba2e-06deb23ca68d)

**Comment by xfgryujk** [comment]
看看服务器日志

**Reply by OcrSu** [reply]
![image](https://github.com/xfgryujk/blivechat/assets/75876201/a03959ca-bd1d-47c4-bfaf-8354ced1e5f4)
![image](https://github.com/xfgryujk/blivechat/assets/75876201/d823097a-b262-4193-87b8-f2209250fdbc)

**Reply by xfgryujk** [reply, marked as answer]
https://github.com/xfgryujk/blivechat/discussions/187

**Reply by OcrSu** [reply]
搞定了，这chat.bilisc.com 默认走的香港代理，我写个直连规则就好了。
怪不得半年前能用，现在回来想用一下就失效了。
谢谢大佬！

**Comment by OcrSu** [comment]
> #187
